src/WalletInit/SaveNanoX/SaveNanoXScreen.tsx

```diff
@@ -30,7 +31,7 @@ export const SaveNanoXScreen = () => {
           readOnly: false,
         })
       }
-      defaultWalletName={CONFIG.HARDWARE_WALLETS.LEDGER_NANO.DEFAULT_WALLET_NAME}
+      defaultWalletName={strings.ledgerWalletNameSuggestion}
```

Comment:
why pass it in as a prop? move it into WalletNameForm?

Reply:
`WalletNameForm` is a shared component between SaveNano and SaveReadOnly components, while refactoring to ts, #1808 , https://github.com/Emurgo/yoroi-mobile/blob/develop/src/WalletInit/SaveReadOnlyWallet/SaveReadOnlyWalletScreen.tsx#L64, it slipped through our fingers, defaultWalletName was in props but not in use.